extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -219,6 +219,14 @@ export class AmpAdXOriginIframeHandler {
       }
     });
 
+    if ((this.element_.attributes.type.nodeValue == 'adsense' ||
```

Comment:
Why check for `adsense` or `doubleclick` here if `lifecycleReporter` already only implements `addPingsForVisibility` for Google implementation?